.github/ISSUE_TEMPLATE/button-request.yml

```diff
@@ -0,0 +1,32 @@
+name: Request a button
+description: Request a button to be added to LinkStack (in the "Pre-defined Sites" block).
+title: "[Button request]: "
+labels: ["button", "enhancement"]
+body:
+- type: input
```

Comment:
Nice one! I'd like to add a checkbox for the user to check if they have searched other issues:

```
- type: checkboxes
    attributes:
      label: Is the button already requested?
      description: Please search to see if an issue for the button already exists.
      options:
      - label: I have searched the existing issues
        required: true
```

Reply:
Totally agree, let's do it!